services/autorust/codegen/src/codegen_operations.rs

```diff
@@ -487,6 +490,18 @@ fn create_function(cg: &CodeGen, doc_file: &Path, operation: &WebOperation) -> R
     Ok(TokenStream::from(func))
 }
 
+fn create_rsp_value(tp: Option<&TokenStream>, fname: &TokenStream) -> TokenStream {
+    if tp.map(|tp| tp.to_string()) == Some("bytes :: Bytes".to_owned()) {
+        quote! {
+            let rsp_value = rsp.body().clone();
```

Comment:
In the rendered output, this provides:

```
let rsp_body = rsp.body();
let rsp_value = rsp.body().clone();
```

Is this intentional?

Reply:
Is it a typo then?  Perhaps this?

```suggestion
            let rsp_value = rsp_body().clone();
```

Reply:
Thanks for catching this. It is resolved.